WooCommerce/Classes/ViewRelated/Dashboard/MyStore/StoreStatsViewController.swift

```diff
@@ -253,6 +253,17 @@ private extension StoreStatsViewController {
             return Constants.quantityDefaultForYear
         }
     }
+
+    func trackStatsLoaded(for granularityString: String) {
```

Comment:
I know this is super nitpicky, but since the dictionary is constructed now within this method, would it make more sense to type the parameter as `StatGranularity` and let the implementation of the method decide what to do with it? (in this case, get its raw value)

Reply:
Good point @ctarda — I will update this!